lib/common/client/concerns/monitoring.rb

```diff
@@ -23,7 +23,8 @@ def increment_total(caller)
         end
 
         def increment_failure(caller, error)
```

Comment:
any chance you could introduce a test for `increment_failure`?

Reply:
yep! I added a test.

Reply:
I'm not seeing any changes to `common/client/concerns/monitoring_spec.rb`, did you push it?

Reply:
oopsie! Now I pushed the commit. Thanks!